consultation_analyser/pipeline/backends/bertopic.py

```diff
@@ -30,6 +30,14 @@ class AnswerWithEmbeddings:
     embedding: list[float]
 
 
+@dataclass
+class AnswerWithEmbeddings2Dim:
```

Comment:
Propose that we don't create a third variation on `Answer` and put the `two_dim_embedding` into the `AnswerWithEmbeddings`, and also assign `two_dim_embedding` at the point of `__get_embeddings_for_question`, merging ` __get_2dim_embeddings` into the body of that method? So every `AnswerWithEmbeddings` has a 2d component